src/hooks/use-ai-agent.ts

```
@@ -0,0 +1,496 @@
+          }
+
+        case 'get_user_info':
+          return JSON.stringify({
+            name: user?.name || 'Unknown',
+            email: user?.email || 'Unknown',
+            school: user?.school || 'Unknown',
+          });
+
+        default:
+          const func =
+            availableFunctions[functionName as keyof typeof availableFunctions];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)